libethereum/EthereumCapability.h

```diff
@@ -200,8 +191,9 @@ class EthereumCapability : public p2p::CapabilityFace, Worker
 
     std::unordered_map<NodeID, EthereumPeer> m_peers;
 
-    std::mt19937_64 m_urng;  // Mersenne Twister psuedo-random number generator
+    std::atomic<bool> m_backgroundWork = {false};
```

Comment:
Can you rename it to `m_backgroundWorkEnabled`?

Reply:
Renamed